jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPopup.java

```diff
@@ -128,6 +127,9 @@ public String getValue(RequestParameterSummary rowObject) {
                                                                                        paramValueColumn));
 
         this.dataProvider.addDataDisplay(executionParametersGrid);
+
+        errorsOccurredList.setVisibleLines(10);
```

Comment:
Would only 10 lines of stack trace be enough to troubleshoot the error?
I guess it's not possible to make the number of lines shown dynamic i.e. to expand to fit the whole stack trace, right?

Reply:
@bsremac I think Neus implemented this correctly. the "visible lines" only refers to the number of rows the textarea will display. It doesn't "trim the stacktrace". The remaining lines are still accessible via textarea's scrollbar as demonstrated [here](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea). I think that it's ok as it is - just show 10 lines and let user resize the area (= default browser behaviour).